Examples/Tests/scraping/analysis_rz_filter.py

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/env python
+
+# Copyright 2019-2021 Yinjian Zhao
```

Comment:
This can be updated with your name and 2023.

Reply:
Done